fxa-auth-db-server/test/backend/db_tests.js

```diff
@@ -1163,6 +1164,57 @@ module.exports = function(config, DB) {
         )
 
         test(
+          'reminders - create and delete',
+          function (t) {
+            t.plan(3)
+            var fetchQuery = {
+              query: {
+                type: 'second',
+                reminderTime: 1,
+                reminderTimeOutdated: 100,
+                limit: 20
+              }
+            }
+            var account = createAccount()
+            return db.createAccount(account.uid, account)
```

Comment:
Possible to add a test case to cover the missing uid/type? https://github.com/mozilla/fxa-auth-db-mysql/blob/db-verification-reminders/lib/db/mysql.js#L551